app/Http/Controllers/RemindersController.php

```diff
@@ -63,4 +66,31 @@ public function update(Reminder $reminder)
 
         return response()->json('updated', Response::HTTP_OK);
     }
+
+    /**
+     * Delete a reminder.
+     *
+     * @param Reminder $reminder
+     * @return JsonResponse
+     * @throws AuthorizationException
+     */
+    public function delete(Reminder $reminder)
+    {
+        $this->authorize('delete', $reminder);
+
+        try {
+            DB::beginTransaction();
+
+            $reminder->delete();
+            Notification::send($reminder->guests, new ReminderDeletedNotification($reminder));
+
+            DB::commit();
+        } catch (\Exception $exception) {
+            DB::rollback();
+
+            return response()->json('There was an error deleting the reminder', Response::HTTP_INTERNAL_SERVER_ERROR);
+        }
+
+        return response()->json('deleted', Response::HTTP_OK);
```

Comment:
This method is unnecessarily complex:
- The try-catch block is not required, since Eloquent throws an `Exception` in case of failure
- The transaction is not required, because a notification won't be sent if the `$reminder->delete()` fails and errors out (it throws an Exception)
- Similarly, only a single `return response()->...` is required. Laravel will return an error AJAX response if it was triggered with an AJAX request. As for the success response, an empty string with "201" would suffice (and stay consistent with the rest of the project)